### ISSUE TITLE
ltc -v and ltc --version regression

### ISSUE THREAD

**Issue by menicosia** [cloudfoundry-incubator/lattice#138]
Looks like `ltc -v` stopped working?
```
$ ./ltc --version
NAME:
   ltc - Command line interface for Lattice.

USAGE:
   ltc [global options] command [command options] [arguments...]

VERSION:
   v0.2.5-27-ga5f57d6

AUTHOR(S):
   Pivotal <cf-lattice@lists.cloudfoundry.org>


COMMANDS:

  TARGET LATTICE:
    target, ta                       Targets a lattice cluster
    target-blob, tb                  Targets a lattice blob store

  CREATE AND MODIFY APPS:
    create, cr                       Creates a docker app on lattice
    remove, rm                       Stops and removes docker app(s) from lattice
    scale, sc                        Scales a docker app on lattice
    update-routes, ur                Updates the routes for a running app

  TASKS:
    submit-task, su                  Submits a task from JSON on lattice
    task                             Displays the status of a given task
    delete-task, dt                  Deletes the given task
    cancel-task, ct                  Cancels the given task

  STREAM LOGS:
    logs, lg, lo                     Streams logs from the specified application

  SEE WHATS RUNNING:
    cells, ce                        Shows details about lattice cells
    list, li, ls                     Lists applications & tasks running on lattice
    status, st                       Shows details about a running app on lattice
    visualize, vz                    Shows a visualization of the workload distribution across the lattice cells

  ADVANCED:
    submit-lrp, create-lrp, cl, sl   Creates a docker app from JSON on lattice
    upload-bits, ub                  Upload bits to the blob store

  HELP AND DEBUG:
    debug-logs, dl                   Streams logs from the lattice cluster components
    test, te                         Runs test suite against targeted lattice cluster
    help, h                          Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --version, -v        Print the version
   --help, -h           Show help
```

**Comment by menicosia** [comment]
@dhilipkumars @ShwethaKumbla, we'd be very happy if you'd like to look at this bug.

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.

**Comment by dhilipkumars** [comment]
Hi @menicosia we will look at this for sure.

**Comment by menicosia** [comment]
Fixed in [v0.2.6](https://github.com/cloudfoundry-incubator/lattice/releases/tag/v0.2.6)! So this never made it to a published release.

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.
